utilities/decrypt_data/decryption_for_program_service.py

```
@@ -0,0 +1,127 @@
+}
+
+# ——— Prepare log directory & filename ——————————————————————————————
+os.makedirs(CONFIG["log_dir"], exist_ok=True)
+ts = datetime.now().strftime("%Y%m%d_%H%M%S")
+logfile = os.path.join(CONFIG["log_dir"],
+                       f"migrate_{CONFIG['table']}_{ts}.log")
+
+logger = logging.getLogger("program_disburse_migrator")
+logger.setLevel(logging.INFO)
+fmt = logging.Formatter("%(asctime)s %(levelname)-8s %(message)s")
+ch = logging.StreamHandler();   ch.setFormatter(fmt); logger.addHandler(ch)
```

Flake8 (7.2.0)

[error] multiple statements on one line (semicolon)

(E702)